components/raftstore/src/store/config.rs

```diff
@@ -573,10 +576,8 @@ impl Config {
                     "store-max-batch-size should be greater than 0 and less than or equal to 10240"
                 ));
             }
-        } else if self.hibernate_regions {
```

Comment:
The default config is changed when hibernate regions is off?

Reply:
ok I'll set 1024 for `hibernate_regions` disabled but config value for enabled.

Reply:
https://github.com/tikv/tikv/pull/12151#discussion_r827610789 here decided to remove it to another PR